uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties

```diff
@@ -18,12 +18,40 @@
 #
 
 
-## TIP:  You can place configuration elements meaningful to your
-## NotificationService implementations in this file and use Maven 
-## filtering to set them.
-
 UsernameFinder.unauthenticatedUsername=guest
 
+finance.endpoint.url=${configuration.finance.endpoint.url}
+finance.endpoint.username=${configuration.finance.endpoint.username}
+finance.endpoint.password=${configuration.finance.endpoint.password}
+
+
+remedyService.username=${configuration.remedyService.username}
+remedyService.password=${configuration.remedyService.password}
+remedyService.locale=${configuration.remedyService.locale}
+remedyService.server=${configuration.remedyService.server}
+remedyService.port=${configuration.remedyService.port}
+remedyService.requestType=${configuration.remedyService.requestType}
+remedyService.baseUrl=${configuration.remedyService.baseUrl}
+
+
+remedyGroupService.username=${configuration.remedyGroupService.username}
+remedyGroupService.password=${configuration.remedyGroupService.password}
+remedyGroupService.locale=${configuration.remedyGroupService.locale}
+remedyGroupService.server=${configuration.remedyGroupService.server}
+remedyGroupService.port=${configuration.remedyGroupService.port}
+remedyGroupService.requestType=${configuration.remedyGroupService.requestType}
+remedyGroupService.baseUrl=${configuration.remedyGroupService.baseUrl}
 
 restfulJsonNotificationService.basicauth.username=CHANGEME
 restfulJsonNotificationService.basicauth.password=CHANGEME
+
+restApi.enabled=false
+
+studentSuccessPlanService.clientId=SETME
+studentSuccessPlanService.clientSecret=SETME
+studentSuccessPlanService.authenticationUrl=/api/1/oauth2/token
+studentSuccessPlanService.sspProtocol=http
+studentSuccessPlanService.sspHost=localhost
+studentSuccessPlanService.sspPort=80
+studentSuccessPlanService.sspContext=ssp
```

Comment:
I think that one is included with the Notification distribution, so I would think "Yes."